File: CPP/Day2/factorial.cpp
#include<iostream>
using namespace std;

int main()
{
    int n, result = 1;
    cout<<"Enter number to find factorial";
    cin>>n;

    if(n==0 || n==1)
    {
        cout<<"Factorial of the given number is 1";
    }
    else if(n<0)
    {
        cout<<"Please enter positive value";
    }
    else
    {
        for(int i=2;i<=n;i++)
        {
            result = result * i; 
        }

        cout<<"Factorial of the given number is "<< result<<endl;
    }
}

File: webtech/Day3/factorial.js
console.log("Hello World");
let fact=1;

function Factorial(num)
{
    for(i=num ;i>=1;i--)
{
    fact = fact*i;
}
}

let num = prompt("Enter Number ");

Factorial(num);
console.log("Factorial of the given number is  - ", fact);
